release-notes/opensearch-release-notes-2.12.0.md

```diff
@@ -0,0 +1,1306 @@
+<h1>OpenSearch and OpenSearch Dashboards 2.12.0 Release Notes</h1>
+<h2>RELEASE HIGHLIGHTS</h2>
+<ul>
+OpenSearch 2.12.0 includes a number of features designed to increase performance for search and analytics applications as well as user experience enhancements and several new and enhanced machine learning tools. Experimental functionality includes the OpenSearch Assistant Toolkit for building interactive user experiences and new cross-cluster monitors. OpenSearch upgrades to Apache Lucene 9.9.1 with this release. 
```

Comment:
lucene version is 9.9.2. Ref [here](https://github.com/opensearch-project/OpenSearch/blob/2.12/buildSrc/version.properties#L2)

Reply:
Updated to 9.9.2.